src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php

```diff
@@ -64,7 +64,17 @@ public function testValidSignature(string $message)
 
         $time = time();
 
-        $lastResponse = new MockResponse();
+        $data = json_encode([
+            'totalCreditsRemoved' => '1',
+            'invalidReceivers' => [],
+            'ids' => [
+                '26929925',
+            ],
+            'validReceivers' => [
+                '+3360000000',
```

Comment:
I think we should match this at least with the number used in SmsMessage above

Reply:
fixed